src/intl/messages.js

```diff
@@ -62,6 +62,7 @@ export const messages: { [messageId: string]: MessageType } = {
   cancel: 'Cancel',
   cannotUpdateCloudInitHostname: 'Host name cannot be synchronized with VM name because the VM name is not a valid host name.',
   cantLogonToConsole: 'Single sign on failed. Please check to see if the guest agent is running on your virtual machine. Contact your administrator if the problem continues.',
+  cantOpenConsole: 'Can\'t open console: {message}',
```

Comment:
I still don't understand why we added this message (Based on Scott's question: "why did we touch the the SSO code that would warrant adding an error toast message?") and anyway it's not informative enough.

Reply:
I thought it will be better if we will notify users if any error occurs during console download...
If it not necessary I'll remove that.

Reply:
You should remove it.  If the SSO doesn't work, a modal is already displayed with an awkward message to handle that scenario.

Right now, on the first run of the app, you'll see both the modal and the toast at the same time.  It isn't necessary.